addOns/encoder/src/main/javahelp/org/zaproxy/addon/encoder/resources/help/contents/encoder.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Advanced Encode / Decode / Hash dialog
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Advanced Encode / Decode / Hash dialog:</H1>
+
+<p>
+This allows you to encode, decode or hash text.
+
+<H2>Common Fields</H2>
+The dialog has one field that is common to all of the tabs:
+
+<H3>Text to be encoded/decoded/hashed:</H3>
+This field is for the text that you want to be encoded, decoded or hashed.<br/>
+If any text is selected when the dialog is invoked then it will be put in this field.
+The other fields will be updated dynamically if you change the text<br/>
+
+<H2>Toolbar</H2>
+
+<ul>
+  <li><img src="images/ui-output--plus.png"></src> Add New Tab - Adds a new tab to the dialog.</li>
+  <li><img src="images/ui-tab--delete.png"></src> Remove Selected Tab - Removes the selected (or current) tab from the dialog.</li>
+  <li><img src="images/ui-tab--plus.png"></src> Add Output Panel to Current Tab - Adds an output panel to the current tab.</li>
+  <li>Reset - Reset all the tabs/panels to their default state.</li>
+</ul>
+
+<H2>Encode tab fields</H2>
+This tab has the following fields:
+
+<H3>Base 64 Encode</H3>
```

Comment:
Should we describe them separately from the default UI and here just mention which ones the default tabs have?
It also allows to describe other encoders that are not in the default UI (script).